packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -385,9 +470,14 @@ export class Select {
 
   private handleSearchableSelectInput = (event: Event): void => {
     this.searchableSelectInputValue = (event.target as HTMLInputElement).value;
+    this.icInput.emit({ value: this.searchableSelectInputValue });
 
-    if (this.getValueFromLabel(this.searchableSelectInputValue) === undefined) {
-      this.icChange.emit({ value: null });
+    if (this.disableFilter === true) {
+      this.emitIcChange(this.searchableSelectInputValue);
+    } else if (
+      this.getValueFromLabel(this.searchableSelectInputValue) === undefined
+    ) {
+      this.emitIcChange(null);
```

Comment:
Don't we still want to emit the value even if there is no matching value form the list? The value has still changed.